### DISCUSSION TITLE
Upload file as binary in POST request

### DISCUSSION THREAD

**Discussion by priyamshah0709** [ovh/venom#698] [Q&A] [answered]
Does anyone have idea, on how to to send a file as binary in the HTTP POST request in venom test yaml. Below i have attached image on how the request looks like in postman and curl, need to replicate that in venom test. 
![Screen Shot 2023-07-10 at 6 53 42 PM](https://github.com/ovh/venom/assets/53808524/16a67237-0b3e-4f06-8fe2-d72f6d0936d6)

**Comment by ivan-velasco** [comment, marked as answer]
You should be able to use `multipart_form` available in the http executor
```yml
multipart_form:
    file: '@filename'
```

**Reply by yesnault** [reply]
yes. And if the executor is missing some options, you can always use a step script with curl to do that.
